### ISSUE TITLE
Open in Chrome for Firefox stopped working

### ISSUE THREAD

**Issue by mikemajara** [andy-portmen/open-in#49]
Comma-separated list of URLs to open with the external browser when the link is opened with left-click¹:
`https://console.aws.amazon.com/`, `https://company.slack.com/`, `https://github.com/org/`

With the above settings. Unchanged since I set up the plugin options, the plugin just stopped sending things to Chrome.

**Comment by andy-portmen** [comment]
I am not able to reproduce this. Have you tried refreshing the source page?

**Comment by MikeMajara** [comment]
Yeah, I tried a bunch of things, I have yet to try downgrading explorer and other stuff. I'll find some time to put together some decent information. Sorry for the brevity.

**Comment by MikeMajara** [comment]
I am using:
- Firefox 84.0.1
- MacOS 10.15.7
- Chrome Version 87.0.4280.88 (Official Build)
- open-in Chrome addon (Firefox -> Chrome)

When debugging the extension I can see the following message in console:

`Unchecked lastError value: Error: Missing host permission for the tab`

Apparently failing at this spot of `background.js`

```
  chrome.tabs.executeScript(tabId, {
    file: "public/js/content-scripts/".concat(contentScriptName),
    matchAboutBlank: true,
    frameId: frameId,
    runAt: 'document_start'
  });
}
```

From the date I realized this was happening (the OP was on Dec 2, 2020), I see that [Chrome released](https://chromereleases.googleblog.com/2020/12/stable-channel-update-for-desktop.html) the very same version I am using.

From the looks Chrome is preventing the add0n to manipulate/create the tab at some point.

Related links:
- https://stackoverflow.com/questions/47503841/error-missing-host-permission-for-the-tab-on-view-source-urls

Let me know If I can do a PR, I am unfamiliar with this projects so it would take me a while still

Thoughts?

**Comment by andy-portmen** [comment]
@MikeMajara are you trying to export an internal tab?

**Comment by MikeMajara** [comment]
@andy-portmen could you elaborate? I am using the same configuration that I was using before... With default settings appart from the URLs marked in "**Comma-separated list of URLs to open with the external browser when the link is opened with left-click1**" (with string e.g.: `https://console.aws.amazon.com/console/home`)

**Comment by MikeMajara** [comment]
I should add to the above that clicking on the web extension button does export the tabs successfully. It is just the automatic opening that is not working as expected.
